catalog/core/catalog-email-impl/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Email Sender"
+         id="org.codice.alliance.core.email.impl.EmailSenderImpl"
+         description="Configure the hostname (or IP address) and port number of an SMTP mail server that will receive the emails. Also, control the maximum total size of attachments for cases where the SMTP server limits the attachment size.">
+
+        <AD
+                description="Mail server hostname (must be resolvable by DNS) or IP address."
+                name="Host" id="mailHost" required="true"
+                type="String" default=""/>
+
+        <AD
+                description="Mail server port number."
+                name="Port" id="mailPort" required="false"
+                type="Integer" default="25"/>
+
+        <AD
+                description="Maximum size of combined attachements (1,000,000 bytes)."
```

Comment:
I think we could probably meet both needs and just say MB and use 1024*1024*number as the input. We should probably default to 25 or 50 as well, since thats a pretty typical max message size from what i've seen.